src/main/java/org/broadinstitute/hellbender/tools/exome/segmentation/AlleleFractionSegmenter.java

```diff
@@ -35,27 +35,23 @@
      */
     public AlleleFractionSegmenter(final int initialNumStates, final AllelicCountCollection acc,
                                    final AllelicPanelOfNormals allelicPoN) {
-        super(initialNumStates, acc.getCounts().stream().map(AllelicCount::getInterval).collect(Collectors.toList()), acc.getCounts());
+        super(initialNumStates, acc.getCounts().stream().map(AllelicCount::getInterval).collect(Collectors.toList()), acc.getCounts(), initialMinorFractions(initialNumStates));
         this.allelicPoN = Utils.nonNull(allelicPoN);
+        biasParameters = new AlleleFractionGlobalParameters(INITIAL_MEAN_ALLELIC_BIAS, INITIAL_ALLELIC_BIAS_VARIANCE, INITIAL_OUTLIER_PROBABILITY);
+
     }
 
     /**
      * evenly-spaced minor allele fractions going from 1/2 to 0
      * @param K the initial number of hidden states
      */
-    @Override
-    protected void initializeHiddenStateValues(final int K) {
-        hiddenStateValues = IntStream.range(0, K).mapToDouble(n ->  ((double) K - n) / (2*K)).toArray();
-    }
-
-    @Override
-    protected void initializeAdditionalParameters() {
-        biasParameters = new AlleleFractionGlobalParameters(INITIAL_MEAN_ALLELIC_BIAS, INITIAL_ALLELIC_BIAS_VARIANCE, INITIAL_OUTLIER_PROBABILITY);
+    private static List<Double> initialMinorFractions(final int K) {
+        return IntStream.range(0, K).mapToDouble(n ->  ((double) K - n) / (2*K)).boxed().collect(Collectors.toList());
```

Comment:
`ScalarHMMSegmenter` assumes that the `NEUTRAL_VALUE_INDEX`, which corresponds to a state that can never be pruned, is `0`, so we need to start with `1/2`.